INFO:uvicorn.error:Started server process [916]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:uvicorn.error:Started server process [995]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [1067]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [1970]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [2564]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [2728]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 139, in GetMindmapCases
    getLastPoint(postTreeData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 199, in getLastPoint
    for j in postTreeData['children']:
TypeError: string indices must be integers

ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 369, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 59, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 52, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 214, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 151, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/concurrency.py", line 40, in run_in_threadpool
    return await loop.run_in_executor(None, func, *args)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 193, in GetMindmapCases
    return templates.TemplateResponse("Error.html")
TypeError: TemplateResponse() missing 1 required positional argument: 'context'
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [2760]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.106:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [05/Jul/2021 23:17:49] "POST /mindmap HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.106:5000/ (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [3033]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [3130]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 141, in GetMindmapCases
    Paths = getPaths(postData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 219, in getPaths
    findPath(data, endpt, path)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 210, in findPath
    if dict_low['id'] == id:
TypeError: string indices must be integers

ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 369, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 59, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 55, in app
    await response(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/templating.py", line 32, in __call__
    extensions = request.get("extensions", {})
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 422, in __get__
    obj = instance._get_current_object()
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 544, in _get_current_object
    return self.__local()  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [3176]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 141, in GetMindmapCases
    Paths = getPaths(postData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 219, in getPaths
    findPath(data, endpt, path)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 210, in findPath
    if dict_low['id'] == id:
TypeError: string indices must be integers

ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 369, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 59, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 55, in app
    await response(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/templating.py", line 32, in __call__
    extensions = request.get("extensions", {})
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 422, in __get__
    obj = instance._get_current_object()
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 544, in _get_current_object
    return self.__local()  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [967]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 142, in GetMindmapCases
    Paths = getPaths(postData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 220, in getPaths
    findPath(data, endpt, path)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 211, in findPath
    if dict_low['id'] == id:
TypeError: string indices must be integers

ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 369, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 59, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 55, in app
    await response(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/templating.py", line 32, in __call__
    extensions = request.get("extensions", {})
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 422, in __get__
    obj = instance._get_current_object()
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 544, in _get_current_object
    return self.__local()  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 142, in GetMindmapCases
    Paths = getPaths(postData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 220, in getPaths
    findPath(data, endpt, path)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 211, in findPath
    if dict_low['id'] == id:
TypeError: string indices must be integers

ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 369, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 59, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 55, in app
    await response(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/templating.py", line 32, in __call__
    extensions = request.get("extensions", {})
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 422, in __get__
    obj = instance._get_current_object()
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 544, in _get_current_object
    return self.__local()  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [1091]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 142, in GetMindmapCases
    Paths = getPaths(postData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 220, in getPaths
    findPath(data, endpt, path)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 211, in findPath
    if dict_low['id'] == id:
TypeError: string indices must be integers

ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 369, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 59, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 55, in app
    await response(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/templating.py", line 32, in __call__
    extensions = request.get("extensions", {})
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 422, in __get__
    obj = instance._get_current_object()
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/werkzeug/local.py", line 544, in _get_current_object
    return self.__local()  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [1314]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:root:Traceback (most recent call last):
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 140, in GetMindmapCases
    Paths = getPaths(postData,endPointDic)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 225, in getPaths
    findPath(data, endpt, path)
  File "/Users/cheuora/works/Vue_Analyzingcontext/backend/main.py", line 216, in findPath
    if dict_low['id'] == id:
TypeError: string indices must be integers

DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [1392]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [1311]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
DEBUG:asyncio:Using selector: KqueueSelector
INFO:uvicorn.error:Started server process [2007]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
